### DISCUSSION TITLE
Warnings for dangerous operations or slider values

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#2346] [Ideas]
It is common knowledge that soft limits or warnings when you do a resource intensive action can go by without any warning with Blender. You could run a SDS level 8, and crash. You could remesh to 0.001 and crash. You can skin a heavy rig and crash. You can skin a heavy mesh and crash. You can move some sliders and values to ridiculous values and crash. 

# Question
As a policy for Bforartists, and an experiment - should we find ways to warn the user about dangerous sliders and operators, explaining a warning before wanting to proceed? Should we add that? 

It is clear blender developers are against this UX.

**Comment by ReinerBforartists** [comment]
Me too. Because this is not to achieve. A laptop might quit with level 2 SDS already.

It should not crash though. That's a bug then. And best reported to Blender devs.

**Reply by Draise14** [reply]
The Voxel Remesh crashing on high density with no soft slider has been reported to Blender and closed often with no solution, it's against their design to not prevent these kinds of user introduced crash (divide by 0 etc). 

If it's resource heavy, it shouldn't crash, but sometimes a resource heavy operation feels like a crash as the computer will hang as it tries to compute. So technically it's not a crash, just.. it hangs calculating because you went too far. 

How could a warning be coded?

**Reply by ReinerBforartists** [reply]
You can't. Same reason as above. A laptop might quit with level 2 sds already :)
